src/openms/source/ANALYSIS/ID/PercolatorFeatureSetHelper.cpp

```diff
@@ -163,7 +163,7 @@ namespace OpenMS
       }
     }
 
-    void PercolatorFeatureSetHelper::addMSFRAGGERFeatures(vector<PeptideIdentification>& peptide_ids, StringList& feature_set)
+    void PercolatorFeatureSetHelper::addMSFRAGGERFeatures(StringList& feature_set)
```

Comment:
use "extra_features" as a variable name and change the ones below?

Reply:
these are not extra features like in the other case but the ones directly returned by MSFragger